viewer/src/main/webapp/flamingo-components/projects/core/src/lib/user-interface/test-attributeservice/test-attributeservice.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import {FormconfigRepositoryService} from "../../shared/formconfig-repository/formconfig-repository.service";
+import {AttributeService} from "../../shared/attribute-service/attribute.service";
+import {AttributeListParameters, RelationType} from "./models";
+
+@Component({
+  selector: 'flamingo-test-attributeservice',
+  templateUrl: './test-attributeservice.component.html',
+  styleUrls: ['./test-attributeservice.component.css']
+})
+export class TestAttributeserviceComponent implements OnInit {
+
+  constructor(
+    private service: AttributeService
+  ) { }
+
+  ngOnInit() {
+  }
+
+  click() {
```

Comment:
@escheper In this method the service is used: an AttributeListParamers object is created with all parameters needed to retrieve the metadata.